package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "scripts": {
+    "clean": "rimraf output && rimraf .pulp-cache",
+    "build": "pulp build -- --censor-lib --strict",
+    "test": "pulp test -- --censor-lib --strict"
```

Comment:
We don't use `--strict` on tests, as the warnings in there don't really matter so much, as they're not part of the published library (the tests sometimes will use `trace`, for example, which now warns on usage)... just `pulp test` will do here!